diff --git a/lib/index.d.ts b/lib/index.d.ts
index ad83aa7253d53652f944f0441eac004a99442cdf..a5f9d7c852aa6eb0339d6e7c3a3233587d13a2a5 100644
--- a/lib/index.d.ts
+++ b/lib/index.d.ts
@@ -2,7 +2,7 @@
 import { Metadata } from './metadata';
 import { Response } from './reader/response';
 import { ReaderOptions } from './types';
-export declare class Reader<T extends Response> {
+export declare class Reader<T> {
     metadata: Metadata;
     private decoder;
     private db;
